alica-plan-designer-fx-codegeneration/src/main/java/de/unikassel/vs/alica/generator/cpp/XtendTemplates.xtend

```diff
@@ -1095,6 +1098,21 @@ namespace alica
 
     }
 
+    «IF (plan.hasDynamicEntryPoint())»
+    bool «plan.name»«plan.id»::getApplicationEntrypointContext(std::unordered_map<int64_t, std::unordered_set<int64_t>>& entryPointMap)
+    {
+        /*PROTECTED REGION ID(getApplicationEntrypointContext«plan.id») ENABLED START*/
+        «IF (protectedRegions.containsKey("getApplicationEntrypointContext" + plan.id))»
+«protectedRegions.get("getApplicationEntrypointContext" + plan.id)»
+        «ELSE»
+            //Add additional options here
```

Comment:
Maybe add log print here like we do for transitions etc suggesting that it is not implemented